plugins/javaembed/javaembed.cpp

```diff
@@ -121,9 +121,13 @@ static class JavaGlobalState
             optionStrings.append(libPath);
         }
 
+        // Options we should set (but allow for override with jvmoptions below)
+        optionStrings.append("-XX:-UseLargePages");
+
         if (conf && conf->hasProp("jvmoptions"))
         {
-            optionStrings.appendList(conf->queryProp("jvmoptions"), ENVSEPSTR);
+            // Use space as field sep as ':' and ';' are valid
+            optionStrings.appendList(conf->queryProp("jvmoptions"), " ");
```

Comment:
This is going to break anyone's existing conf file settings, so will require a mention in the redbook at least